### ISSUE TITLE
feat(website): add link to Discussions on github?

### ISSUE THREAD

**Issue by dantaik** [taikoxyz/taiko-mono#403]
### Describe the feature request

[Description of the feature request here.https://github.com/taikoxyz/taiko-mono/discussions](https://github.com/taikoxyz/taiko-mono/discussions)

As of now the above discussion board is not active at all, I think we should promote the link on our website and also in our discord server.

**Comment by 1xDeFi** [comment]
Yes, in addition to the website. It should be added to the discord official-links channel

**Comment by d1onys1us** [comment]
> Yes, in addition to the website. It should be added to the discord official-links channel

@odesium maybe a good time to use the carl-bot and add it to official-links

**Comment by d1onys1us** [comment]
added here https://github.com/taikoxyz/taiko-mono/pull/436

**Comment by odesium** [comment]
> > Yes, in addition to the website. It should be added to the discord official-links channel
> 
> @odesium maybe a good time to use the carl-bot and add it to official-links

Added to Discord for more visability in Official Links.
